portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -54,8 +54,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       String body = (String) entity.getProperty("body");
       long timestamp = (long) entity.getProperty("timestamp");
       String email = (String) entity.getProperty("email");
+      float score = ((Double) entity.getProperty("score")).floatValue();
```

Comment:
This seems like a lot of unnecessary casting. Does `float score = entity.getProperty("score");` not work? If not, then it would be nicer to just use double everywhere.

Reply:
* `entity.getProperty(score)` doesn't work

*  If I use double for score then sometimes the score has a lot of decimals
    - I guess I could set precision to fewer decimal places when I display the score?

Reply:
![image](https://user-images.githubusercontent.com/41055600/88779307-4c138700-d192-11ea-9660-57672deb0af0.png)